applet/src/os.c

```diff
@@ -101,6 +101,7 @@ void * OSMboxPend_hook(OS_EVENT *pevent, uint32_t timeout, int8_t *perr)
                 last_radio_event = *(uint8_t*)ret ;
             }
             if( ((uint32_t)pevent) == 0x20017390 ) {
+                // beep events?
```

Comment:
i have located the mbox pointer on create mbox and verified   on the calling function